plugins/module_utils/openssh/backends/keypair_backend.py

```diff
@@ -0,0 +1,478 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2021, Andrew Pantuso (@ajpantuso) <ajpantuso@gmail.com>
```

Comment:
Two quick comments on this file:
1. The reason other modules have most of their code in module_utils is that the code is used by multiple modules. Do you plan do add a new module that also uses this code? (If not, you this refactoring is also ok, just not really needed :) )
2. Since you probably took some code from the main module, you should include it's [copyright notice](https://github.com/ansible-collections/community.crypto/blob/83f01ab2fb88b959e125a3cc91b5aa15da3242f7/plugins/modules/openssh_keypair.py#L4) here as well.

Reply:
1. No not in the immediate future (though I was considering some filters or lookups for OpenSSH public keys). I had originally planned on just passing the `backend` to the main class and toggling internally, but this module was clunky enough already. So if I wanted it to be more readable/maintainable I figured following a consistent pattern from the collection was probably a good start even if not necessary.
2. Just added; thanks for catching that.